### DISCUSSION TITLE
Program won't start

### DISCUSSION THREAD

**Discussion by ZaZaNonStop** [xenia-manager/xenia-manager#105] [Q&A] [answered]
When I downloaded xenia manager, I can't get it to launch after double clicking the app. Xenia works fine but manager isn't working for me.

**Comment by shazzaam7** [comment, marked as answer]
You can report bugs in the Issues section. Please fill it out accordingly otherwise it will just get closed.

**Reply by ZaZaNonStop** [reply]
Thank you.
